Chummer/Classes/clsXmlManager.cs

```diff
@@ -1857,6 +1857,69 @@ public bool Equals(CustomDataDirectoryInfo other)
             return other != null && Name == other.Name && Path == other.Path;
         }
 
+        /// <summary>
+        /// Reads in an Manifest.xml in the given directory and returns it's language specific description. Defaults to an description without any attributes.
+        /// </summary>
+        /// <param name="strDirectory"></param>
+        /// <returns>A description as String, adds a comment if it defaulted to a node without language attribute</returns>
+        public string GetManifestData(string strDirectory)
+        {
+            string strLanguage = GlobalOptions.Language;
+            XmlDocument xmlObjManifest = new XmlDocument();
+            string strFullDirectory = strDirectory + "\\manifest.xml";
+            bool blnDefaultedToEng = false;
+
+            //This will come up very often, so check this here instead of handling the exception. It also catches all exceptions regarding permission and bad strings(Paths)
+            if(!File.Exists(@strFullDirectory))
+                return LanguageManager.GetString("Tooltip_CharacterOptions_ManifestMissing");
```

Comment:
It might be annoying to have "manifest missing" pop up as tooltips for most stuff. Maybe it would be better to have tooltips be blank for stuff without any manifest file at all, at least if you stay with the current setup.

Reply:
For me it was fine, but I can see how this is annoying to other users.
Gonna take this out and just return from the method.